OrcScala/src/orc/values/sites/OrcJavaCompatibility.scala

```diff
@@ -75,9 +75,11 @@ object OrcJavaCompatibility {
     def getParameterTypes(): Array[java.lang.Class[_]]
     def isStatic: Boolean
     def isVarArgs: Boolean
+    def getName(): String
     def invoke(obj: Object, args: Array[Object]): Object
   }
 
+  // TODO: Fix spelling on invo(c|k)able.
```

Comment:
"Invocable" is the standard English spelling, and the spelling used for javax.script.Invocable, for example.